### ISSUE TITLE
Values with whitespace can not be used in where clause

### ISSUE THREAD

**Issue by giorgostheo** [DataStories-UniPi/miniDB#151]
It seems that when the value of a column contains whitespaces, it can not be used inside the where clause. Example below. 

This should be fixed using "" that group a string together in mdb's sql compiling phase. The easier way would be to just replace whitespace with _, but that would not be consistent with other rDBMSs.

![image](https://user-images.githubusercontent.com/15364873/172822021-c4387703-f1a9-44ec-82e2-c2e4a29de33e.png)

**Comment by Stratis-Dermanoutsos** [comment]
The cause of this is in misc.py, line 19, where all whitespaces are deleted.

```python
def split_condition(condition):
    print(f'condition before: {condition}') #!! this is fine
    condition = condition.replace(' ','') # remove all whitespaces
    print(f'condition after: {condition}') #!! this is not fine
    ops = {'>=': operator.ge,
           '<=': operator.le,
           '=': operator.eq,
           '>': operator.gt,
           '<': operator.lt}

    for op_key in ops.keys():
        splt=condition.split(op_key)
        print(f'splt: {splt}')
        if len(splt)>1:
            return splt[0], op_key, splt[1]
```

I am working on a fix to leave the value as is.

**Comment by Stratis-Dermanoutsos** [comment]
It has been solved in #152.
I believe this issue can close now.
